services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -373,6 +373,8 @@ public class VeniceHelixAdmin implements Admin, StoreCleaner {
 
   private DataRecoveryManager dataRecoveryManager;
 
+  private boolean isLeaderReplicaFailOverEnabled;
```

Comment:
Can we rename it to be `isErrorLeaderReplicaFailOverEnabled` to be consistent with the config option?

Reply:
sure